style.css

```diff
@@ -0,0 +1,118 @@
+@import "https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap";
+
+:root {
+  --main-bg-color: #fff;
+  --normal-text-color: #344563;
+  --gray-background-color: #fff;
+  --container-bg: #e5e5e5;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  margin: 0;
+  padding: 0;
+  overflow-x: hidden;
+  overflow-y: auto;
+  outline: none;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+li {
+  list-style: none;
+}
+
+.container {
+  width: 380px;
+  margin: 0 auto;
+  border: 1px solid #7f8cff;
+  background-color: var(--container-bg);
+}
+
+header {
+  width: 376px;
+  height: 48px;
+  top: 44px;
+  padding: 3%;
+  background-color: var(--main-bg-color);
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  padding: 0.5 4%;
+}
+
+.logo {
+  display: flex;
+  align-self: flex-end;
+}
+
+.small-container > h1 {
+  font-size: 2.5em;
+  padding-top: 0.5rem;
+  font-weight: 700;
+  line-height: 3.6rem;
+  font-style: normal;
+  color: #172b4d;
+}
```

Comment:
- [ ] **Good job on making a good design 🎖️** however, It would be better to add the proper margins (top, bottom, left, right) defined in the design 👍 

   _you may follow this [link](https://www.geeksforgeeks.org/css-margins-padding/) for more info on Margins and Padding_ 


## Figma design
![image](https://user-images.githubusercontent.com/102757126/211599141-e710b990-222e-44d5-abe1-47427cc74ee3.png)

## Your design
![image](https://user-images.githubusercontent.com/102757126/211600035-502d211f-2708-4f06-a7f5-fa1bbd661186.png)